EventsExpress/ClientApp/src/services/EventsExpressService.js

```diff
@@ -288,6 +288,49 @@ export default class EventsExpressService {
         return res;
     }
 
+    setEvent = async (data) => {
```

Comment:
There was one method AFAIR for saving events. Why a new one?

Reply:
There is only one method for saving events, i didn't do more